### PR TITLE
W-10673246 adding content for running mutations

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 1]
```diff
@@ -1,15 +1,15 @@
-= Querying the Unified Schema
+= Running Queries and Mutations
 
-The unified schema resolves into a single GraphQL endpoint. To query the unified schema, you first request access (if applicable) and then write and run a GraphQL query in the query editor.
+The unified schema resolves into a single GraphQL endpoint. To run queries or mutations against the unified schema, you first request access (if applicable) and then write and run a GraphQL query in the query editor.
 
 == Prerequisites
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:5]: == Before You Begin

File: modules/ROOT/pages/datagraph-qsg.adoc [hunk 2]
```diff
@@ -199,7 +199,7 @@ The results show that you’re performing a local merge between the OrderSummary
 
 As Anypoint DataGraph updates the unified schema, you can navigate through the schema to view the changes you just made. When the status changes to “Up to date”, indicating that the unified schema has been updated with your changes, proceed to the next step.
 
-== Request Access to Query the Unified Schema
+== Request Access to Run Queries and Mutations
 
 After you add the two API schema’s to the unified schema, you’re ready to request permission for access to run a query.
```
[the remark this answers is not in the document]
Comment (kbostrom) [reply modules/ROOT/pages/datagraph-qsg.adoc:204]: What about the three types of permissions?

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to use request access and run queries.
+* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to use request access and run queries and mutations.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:7]:
```suggestion
* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission]. 
```
We don't _have_ to tell _why_ in a prereq. list item. Your choice. If you keep...:
```suggestion
* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to run queries and mutations.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 4]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
 * Duration of requests to each source API in the query
 
-. To see response logs associated with the query, click *View response logs*.
+. To see response logs associated with the query, from the actions menu (*...*), select *View Response Logs*.
 +
 image::datagraph-qsg-response-logs.png[Query response logs page]
 +
-Log levels for Anypoint DataGraph include, DEBUG, INFO, WARN, and ERROR.
+Log levels for  DataGraph include, DEBUG, INFO, WARN, and ERROR.
 . Click *View History* to access this same query—or others—at a later time.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:67]:
```suggestion
. Click *View History* to access this same query (or others) later.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 5]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
 . To use this query in your application, click *Copy endpoint* to copy the query and the automatically generated GraphQL endpoint. You can also copy the query as a cURL snippet.
 
+== Run a Mutation
+
+Use the query editor to write, run, and troubleshoot mutations.
+
+. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers*, and:
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:74]:
```suggestion
. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers* and:
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 6]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
+
+.. Select a header name.
+.. Enter a header value.
+.. To specify additional header name and value pairs for other APIs, click *+Add Header* x
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:78]: x?
```suggestion
.. To specify additional header name and value pairs for other APIs, click *+Add Header*.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 7]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
+
+. If any of the APIs in the unified schema require pass-through authentication, click *Add HTTP Headers*, and:
+
+.. Select a header name.
+.. Enter a header value.
+.. To specify additional header name and value pairs for other APIs, click *+Add Header* x 
+.. Click *Save*.
+.. To edit the HTTP headers, click *Edit HTTP Headers*.
+
+. In the query editor, start typing your mutation using the in-line autocomplete function or type *Option/Alt + Shift + Space* to open the autocomplete menu.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:82]:
```suggestion
. In the query editor, start typing your mutation using the in-line autocomplete function, or type *Option/Alt + Shift + Space* to open the autocomplete menu.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 8]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
++
+image::datagraph-mutation-autocomplete.png[Using inline autocompletion to write a mutation,70%]
+
+. Optionally, in the **Query Variables** pane, add any variables you want to use.
+. Click **Run**.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:87]: Double *s okay?

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 9]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
+
+. In the query editor, start typing your mutation using the in-line autocomplete function or type *Option/Alt + Shift + Space* to open the autocomplete menu.
++
+image::datagraph-mutation-autocomplete.png[Using inline autocompletion to write a mutation,70%]
+
+. Optionally, in the **Query Variables** pane, add any variables you want to use.
+. Click **Run**. 
++
+image::datagraph-mutation-result.png[Query editor displays mutation results]
+. To see results with query tracing, from the actions menu (*...*), select *Trace Query*, and then click *Run*.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:90]:
```suggestion
. To see results with query tracing, from the actions menu (*...*), select *Trace Query* and then click *Run*.
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 10]
```diff
@@ -49,24 +49,49 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
+. To see response logs associated with the query, from the actions menu (*...*), select *View Response Logs*.
+. Click *View History* to access this same mutation—or others—at a later time.
```
Comment (kbostrom) [inline modules/ROOT/pages/query-unified-schema.adoc:92]:
```suggestion
. Click *View History* to access this same mutation (or other) later.
```

**Review by kbostrom — DISMISSED** [review]
quick edit

File: modules/ROOT/pages/query-unified-schema.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/query-unified-schema.adoc:5]:
```suggestion
== Before You Begin
```

File: modules/ROOT/pages/query-unified-schema.adoc (hunk 8 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/query-unified-schema.adoc:87]:
```suggestion
. Click *Run*. 
```

File: modules/ROOT/pages/query-unified-schema.adoc [hunk 11]
```diff
@@ -49,23 +49,48 @@ image::datagraph-query-variable.png[Inserting a query variable in the query edit
...
-Log levels for Anypoint DataGraph include, DEBUG, INFO, WARN, and ERROR.
-. Click *View History* to access this same query—or others—at a later time.
-. To use this query in your application, click *Copy endpoint* to copy the query and the automatically generated GraphQL endpoint. You can also copy the query as a cURL snippet.
+Log levels for DataGraph include DEBUG, INFO, WARN, and ERROR.
+. Click *View History* to access this same query (or others) later.
+. To use this query in your application, click *Copy endpoint* to copy the query and the automatically generated GraphQL endpoint. You can also copy the query as a curl snippet.
+
+== Run a Mutation
+
+Use the query editor to write, run, and troubleshoot mutations.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/query-unified-schema.adoc:72]:
```suggestion
Use the query editor to write and then run mutations. If needed, you can also troubleshoot mutations in the query editor
```

**Review by tfoxy — APPROVED** [review]

**Pull request merged**